kazoo/client.py

```diff
@@ -79,15 +79,13 @@
     max_retries=None,
     retry_delay=0.1,
     retry_backoff=2,
-    retry_jitter=0.8,
     retry_max_delay=3600,
 )
 
 _RETRY_COMPAT_MAPPING = dict(
     max_retries='max_tries',
     retry_delay='delay',
     retry_backoff='backoff',
-    retry_jitter='max_jitter',
```

Comment:
Note: Technically this is a backwards-incompatible change as it will stop warning that this is a deprecated param. However, it looks like this has been deprecated for over 5 years since version 1.2: https://github.com/python-zk/kazoo/commit/3d577e4f4167d72e0e1553837b602ac2da2848d0#diff-7d5ec76d88673978d0b9b679863f349dR228

So I'm in agreement with simply removing this altogether.